src/main/scala/com/gd/twitteranalytics/TweetsDataSave.scala

```diff
@@ -0,0 +1,13 @@
+package com.gd.twitteranalytics
+
+import org.apache.spark.sql.DataFrame
+import com.gd.twitteranalytics.util.ReadTwitterConf
+
+object TweetsDataSave {
+
+  def saveOutputToHdfs(inputDataFrame:DataFrame)={
```

Comment:
To make the API testable and more self-descriptive, any environment dependent parameters (e.g. save-path) should be passed as arguments to the function.

Reply:
Fixed !